polar_route/route_planner/route_planner.py

```diff
@@ -301,11 +309,15 @@ def _splitting_around_waypoints(self, waypoints_df):
         """
         if ('waypoint_splitting' in self.config) and (self.config['waypoint_splitting']):
             logging.info(' Splitting around waypoints !')
+            prior_cells = [cellbox.get_id() for cellbox in self.env_mesh.agg_cellboxes if cellbox.agg_data['inaccessible']]
```

Comment:
Should this be `if not cellbox.agg_data['inaccessible']`? (same on line 315 below)

Reply:
good spot, that shouldn't be the case. Will correct now and rerun tests now.

Reply:
Fixed, and re-ran the regression tests. 
[regression_tests.txt](https://github.com/user-attachments/files/17586587/regression_tests.txt)